man/docker-attach.1.md

```diff
@@ -18,15 +19,17 @@ interactively.  You can attach to the same contained process multiple times
 simultaneously, screen sharing style, or quickly view the progress of your
 detached process.
 
-You can detach from the container (and leave it running) with `CTRL-p CTRL-q`
-(for a quiet exit) or `CTRL-c` which will send a `SIGKILL` to the container.
-When you are attached to a container, and exit its main process, the process's
-exit code will be returned to the client.
+To stop a container, use `CTRL-c`. This key sequence sends `SIGKILL` to the
+container. You can detach from the container (and leave it running) using a
+configurable key sequence. The default sequence is `CTRL-p CTRL-q`.
```

Comment:
```Markdown
To stop a container, use `CTRL-c`. This key sequence sends `SIGKILL` to the
 container. You can detach from the container (and leave it running) using a
 configurable key sequence. You configure the key sequence using the
`--detach-keys` option. The default sequence is `CTRL-p CTRL-q`, and
can be changed using a configuration file. See **docker(1)** for documentation
on using a config file.
```

Hm, only it looks like the long description of "config files" is not in the man page :/